### DISCUSSION TITLE
What does V2ray.exe do?

### DISCUSSION THREAD

**Discussion by Persianfriend** [MortezaBashsiz/CFScanner#132] [Q&A] [answered]
ضمن تشکر بابت این برنامه فوق‌العاده، 

در صورت امکان لطفا کاربرد و نحوه استفاده از فایل V2ray.exe که در فولدر برنامه موجود هست رو با ذکر یک مثال مختصرا توضیح بدید. 🙏🏻

سپاس فراوان

**Comment by goingfine** [comment, marked as answer]
نیاز به اجرای اون فایل ندارید

**Reply by Persianfriend** [reply]
سپاس فراوان از پاسخگویی شما
